dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -680,7 +680,7 @@ class ScriptsControllerTest < ActionController::TestCase
     }
 
     assert_response :not_acceptable
-    msg = 'Legacy script levels are not allowed in migrated scripts. Problem lessons: [\"problem lesson\"]'
+    msg = 'Legacy script levels are not allowed in migrated units. Problem lessons: [\"problem lesson\"]'
```

Comment:
just noting that it's unclear how soon we'll be able to rename "script levels", so i think it's good that you're leaving those out of this for now. I haven't wrapped my mind around whether we want to rename those to "unit levels" or not.

Reply:
Ya I've been intentional avoiding script levels but did have a similar wonder if we will leave them named as is or update